docs/hopr-documentation/docs/developers/connecting-node.md

```diff
@@ -0,0 +1,135 @@
+---
+id: connecting-node
+title: Interacting with a HOPR node
+---
+
+Once you have your HOPR cluster up and running, you are ready to connect to a single HOPR node and listen to messages sent to it by other HOPR nodes.
+The following guide will show you how to connect to a running HOPR node, and verify some basic functionality using both the REST and WebSocket
+endpoints.
+
+## Connect to a HOPR node
+
+### 1. Get the node security credentials
+
+To avoid unsecured access to your HOPR node, all HOPR node's WebSocket and REST calls require an `apiToken`. Your API token needs to be appended as a
+`query` parameter for WebSocket connections and as encoded token via the `Authorization` header for you to be able to connect to it.
+
+:::info Tip
+The flag used to set this value in a HOPR node via `hoprd` is `--apiToken`. The default `apiToken` used across our documentation is the following one 
+(click on the right to copy), so make sure to change it when running your own node in a public network.
+<br/>
+
+```
+^^LOCAL-testing-123^^
+```
+:::
+
+Export your `apiToken` to be used in the incoming commands.
+
+```
+export apiToken="^^LOCAL-testing-123^^"
+```
+
+### 2. Test REST API connectivity
+
+Given your node's `apiToken` and your `HOPR_NODE_1_HTTP_URL` from the last section, try to send a REST command to query it’s address with the following `curl`
+command. If you don’t have `jq` installed, just remove it at the end of the command.
+
+```
+echo -n $apiToken | base64 | xargs -I {} curl -s -H "Authorization: Basic {}" $HOPR_NODE_1_HTTP_URL/api/v2/account/address | jq
+```
+
+If successful, you should get a response similar to this one:
+
+```
+{
+  "nativeAddress": "0x3a54dDE3ee5ACfd43C902cbecC8ED0CBA10Ff326",
+  "hoprAddress": "16Uiu2HAmE9b3TSHeF25uJS1Ecf2Js3TutnaSnipdV9otEpxbRN8Q"
+}
+```
+
+In case you have made a mistake, like forgotten to use `-n` in your `echo` or have the wrong `apiToken`, you’ll see the following instead:
+
+```
+{
+  "status": 403,
+  "challenge": "Basic realm=hoprd",
+  "message": "You must authenticate to access hoprd."
+}
+```
+
+### 3. Test WebSocket connectivity
+
+Unlike our REST API endpoint, to see interactions against our HOPR node WebSocket server, we need to use a client to both send and listen
+connections in our HOPR node. Our suggested client is [websocat](https://github.com/vi/websocat), which you can install by running our
+`./scripts/install-websocat.sh` [script](https://raw.githubusercontent.com/hoprnet/hoprnet/master/scripts/install-websocat.sh) from our monorepo.
```

Comment:
Maybe a note that this script installs `websocat` to `../.bin` directory relative to `./scripts`.

Reply:
Yeah, that's really annoying, but noted.

Reply:
```suggestion
`./install-websocat.sh` [script](https://raw.githubusercontent.com/hoprnet/hoprnet/master/scripts/install-websocat.sh) from our monorepo.
To install, make sure to run it from the `scripts` folder, as it will install it in the parent folder under a `.bin` folder, and will not export it to your `$PATH`.
```